libs/wingc/src/type_check/jsii_importer.rs

```diff
@@ -566,41 +562,36 @@ impl<'a> JsiiImporter<'a> {
 		// Get the base class of the JSII class, define it via recursive call if it's not define yet
 		let base_class_type = if let Some(base_class_fqn) = &jsii_class.base {
 			let base_class_fqn = FQN::from(base_class_fqn.as_str());
-			// Hack: if the base class name is a resource base then we treat this class as a resource and don't need to define its parent.
-			if is_construct_base(&base_class_fqn) {
-				is_resource = true;
-				None
```

Comment:
Can you explain what happens when we import something that derives from `constructs.Construct`. Does it import `constructs.Construct`? Does it also import `constructs.IConstruct`? Can we close #261?

Reply:
Yep, we are importing `constructs.Construct` and `constructs.IConstruct` now. Thanks for bringing that issue to my attention